test/base_test.rb

```diff
@@ -65,6 +65,10 @@ def method_missing(symbol, *arguments, &block)
         user
       end
     end
+
+    def build_ar_parameters(hash)
+      ActionController::Parameters.new(hash)
+    end
```

Comment:
This method is being repeated a lot, may we put it on `test_helper`, inside `ActiveSupport::TestCase`?

Reply:
Sure, that's fine. I don't mind the repetition either. I'll leave that up to you.

Can you please rename the method to just `build_parameters`?

Reply:
Sure, and I left the tests self-contained, thanks for the feedback.